manifests/generate_archive.pp

```diff
@@ -0,0 +1,46 @@
+# Prepare the certificates for the node from the parent node
```

Comment:
Could you maintain backwards compatibility for the time being by having the foreman_proxy_content class call this class with parameters?